src/components/molecules/RadioGroupField/RadioGroupField.tsx

```diff
@@ -14,10 +14,9 @@ const RadioWrapper = styled.div`
 
 type RadioGroupFieldItem = {
   value: string;
-  label?: string;
+  label?: string | React.ReactNode;
```

Comment:
`React.ReactNode` already accepts `string`

Reply:
🤦‍♀️ Should I open a pr with a fix or is it not worth it for such a small thing and leave it to be fixed in another pr?

Reply:
It's not worth it. I commented it just to let you know :) It can be fixed in another PR